privacyidea/static/components/user/views/user.html

```diff
@@ -4,7 +4,7 @@
         <div class="col-lg-3">
             <div class="pa-sidebar panel">
                 <ul class="nav nav-pills nav-stacked"
-                    ng-show="loggedInUser.role === 'user'">
+                    ng-show="loggedInUser.role == 'user' && checkRight('updateuser') && checkRight('password_reset')">
```

Comment:
Hi Cornelius,

No problem, I remove this too.

Best Regards

Martin

On 2016-03-24 19:32, Cornelius Kölbel wrote:
> In privacyidea/static/components/user/views/user.html [1]:
> 
>> @@ -4,7 +4,7 @@
>>          <div class="col-lg-3">
>>              <div class="pa-sidebar panel">
>>                  <ul class="nav nav-pills nav-stacked"
>> -                    ng-show="loggedInUser.role === 'user'">
>> +                    ng-show="loggedInUser.role == 'user' &&
> checkRight('updateuser') && checkRight('password_reset')">
> 
> updateuser is fine.
> But you need to remove password_reset as mentioned in the comment
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub [2]
> 
> 	*
> 
> Links:
> ------
> [1] 
> https://github.com/privacyidea/privacyidea/pull/345#discussion_r57374857
> [2]
> https://github.com/privacyidea/privacyidea/pull/345/files/f64bfe1a658f9e19818e9a58430fb91b1a3868f5#r57374857